diff --git a/ptool.py b/ptool.py
index 031ad74..e4465a7 100644
--- a/ptool.py
+++ b/ptool.py
@@ -682,6 +682,8 @@ def CreateGPTPartitionTable(PhysicalPartitionNumber,UserProvided=False):
             Attributes |= 1<<62
         if PhyPartition[k][j]['dontautomount']=="true":
             Attributes |= 1<<63
+        if PhyPartition[k][j]['uc20_install_created']=="true":
+            Attributes |= 1<<59
         if PhyPartition[k][j]['system']=="true":
             Attributes |= 1<<0
         if PhyPartition[k][j]['tries_remaining']>0:
@@ -1269,6 +1271,8 @@ def ParseXML(XMLFile):
                 Partition['sparse']              = ["false"]
                 Partition['filepartitionoffset'] = [0]
 
+                Partition['uc20_install_created'] = ["false"]
+
                 Partition['size_in_kb']         = 0
                 Partition['original_size_in_kb']= 0
                 Partition["readonly"]           = "false"
